packages/manager/src/components/TableSortCell/TableSortCell.tsx

```diff
@@ -39,7 +46,14 @@ export const TableSortCell = (props: TableSortCellProps) => {
 
   return (
     <TableCell
+      aria-sort={ariaSort}
       onClick={onHandleClick}
+      onKeyDown={(e) => {
+        if (e.key === 'Enter' || e.key === ' ') {
+          e.preventDefault();
+          onHandleClick?.();
```

Comment:
```suggestion
          onHandleClick();
```
Do we need the optional chaining operator here? onHandleClick is defined above.

Reply:
You're right, not needed. I've removed it. Thanks!